src/App.css

```diff
@@ -111,6 +115,110 @@
 	transform: translate(-5%, 10%);
 }
 
+.loader-container {
+	display: none;
+	justify-content: center;
+	align-items: center;
+	position: absolute;
+	z-index: 999;
+	top: 50%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+	background: #000;
```

Comment:
I think without this background it would be more sick! ;)

Reply:
Got it !